plugin/lsp.vim

```diff
@@ -26,6 +26,10 @@ def g:LspServerReady(): bool
   return lsp.ServerReady()
 enddef
 
+def g:LspServerStatus(ftype: string): bool
```

Comment:
As this function returns the server running status, can you change the name of this function to LspServerRunning?